### PR TITLE
Give a higher imp to words in the title

### PR DESCRIPTION
**Pull request by loicgasser**
This will affect the ranking in the layer search.

This is to prioritize titles over description fields works with:
https://github.com/geoadmin/service-sphinxsearch/pull/256
and fixes:
https://github.com/geoadmin/mf-chsdi3/issues/1709

### CODE REVIEW

File: chsdi/views/search.py [hunk 1]
```diff
@@ -167,6 +168,13 @@ def staging_filter(staging):
         self.sphinx.SetLimits(0, layerLimit)
         self.sphinx.SetRankingMode(sphinxapi.SPH_RANK_WORDCOUNT)
         self.sphinx.SetSortMode(sphinxapi.SPH_SORT_EXTENDED, '@weight DESC')
+        # Weights defaults to 1
+        self.sphinx.SetFieldWeights({
+            '@title': 4,
+            '@detail': 2,
+            '@layer': 1
```
Comment (gjn) [inline chsdi/views/search.py]: I would switch weights for @layer and @detail.

Comment (loicgasser) [inline chsdi/views/search.py]: layer is the technical name, that's for us only and if you have the technical name well you usually only have one result. but I can change it.

Comment (loicgasser) [inline chsdi/views/search.py]: @gjn still want me to change it?

**Comment by gjn** [comment]
We keep it as it is. But we merge this not for tomorrow's deploy.

**Pull request merged**